pkg/network/node/ovscontroller.go

```diff
@@ -494,31 +493,34 @@ func (oc *ovsController) UpdateEgressNetworkPolicyRules(policies []networkapi.Eg
 	errs := []error{}
 
 	if len(policies) == 0 {
-		otx.DeleteFlows("table=101, reg0=%d", vnid)
+		otx.DeleteFlows("table=100, reg0=%d", vnid)
 	} else if vnid == 0 {
 		errs = append(errs, fmt.Errorf("EgressNetworkPolicy in global network namespace is not allowed (%s); ignoring", policyNames(policies)))
 	} else if len(namespaces) > 1 {
 		// Rationale: In our current implementation, multiple namespaces share their network by using the same VNID.
 		// Even though Egress network policy is defined per namespace, its implementation is based on VNIDs.
 		// So in case of shared network namespaces, egress policy of one namespace will affect all other namespaces that are sharing the network which might not be desirable.
 		errs = append(errs, fmt.Errorf("EgressNetworkPolicy not allowed in shared NetNamespace (%s); dropping all traffic", strings.Join(namespaces, ", ")))
-		otx.DeleteFlows("table=101, reg0=%d", vnid)
-		otx.AddFlow("table=101, reg0=%d, priority=1, actions=drop", vnid)
+		otx.DeleteFlows("table=100, reg0=%d", vnid)
+		otx.AddFlow("table=100, reg0=%d, priority=101, actions=drop", vnid)
 	} else if len(policies) > 1 {
 		// Rationale: If we have allowed more than one policy, we could end up with different network restrictions depending
 		// on the order of policies that were processed and also it doesn't give more expressive power than a single policy.
 		errs = append(errs, fmt.Errorf("multiple EgressNetworkPolicies in same network namespace (%s) is not allowed; dropping all traffic", policyNames(policies)))
-		otx.DeleteFlows("table=101, reg0=%d", vnid)
-		otx.AddFlow("table=101, reg0=%d, priority=1, actions=drop", vnid)
+		otx.DeleteFlows("table=100, reg0=%d", vnid)
+		otx.AddFlow("table=100, reg0=%d, priority=101, actions=drop", vnid)
 	} else /* vnid != 0 && len(policies) == 1 */ {
-		otx.DeleteFlows("table=101, reg0=%d", vnid)
+		otx.DeleteFlows("table=100, reg0=%d", vnid)
 
 		for i, rule := range policies[0].Spec.Egress {
-			priority := len(policies[0].Spec.Egress) - i
+			// The priority here needs to be higher than the egressIPPriority
+			// for the rules to be evaluated in the order: egress network policy
+			// -> egress IP
+			priority := len(policies[0].Spec.Egress) - i + egressIPPriority
```

Comment:
no... ENPs and egress IPs are in different tables. You shouldn't have to change anything with priority